packages/core/src/codewhisperer/service/transformByQ/transformationResultsViewProvider.ts

```diff
@@ -423,8 +423,8 @@ export class ProposedTransformationExplorer {
                             programmingLanguage: {
                                 languageName:
                                     transformByQState.getTransformationType() === TransformationType.LANGUAGE_UPGRADE
-                                        ? 'JAVA'
-                                        : 'SQL',
+                                        ? 'java'
```

Comment:
If there is potential that these values will be used in other telemetry events this may be worth putting in to a function

Reply:
Yeah, post-re:Invent planning on adding a separate utils file for these kinds of things